operator/api/v1alpha1/kyma_types.go

```diff
@@ -342,57 +322,46 @@ func init() {
 	SchemeBuilder.Register(&Kyma{}, &KymaList{})
 }
 
-const NewModuleMessage = "new module"
-
-func (kyma *Kyma) MatchConditionsToModules() {
-	for _, module := range kyma.Spec.Modules {
-		found := false
-		for _, condition := range kyma.Status.Conditions {
-			if module.Name == condition.Reason {
-				found = true
-			}
-		}
-		if !found {
-			newCondition := KymaCondition{
-				Type:    ConditionTypeReady,
-				Status:  ConditionStatusFalse,
-				Reason:  module.Name,
-				Message: NewModuleMessage,
-			}
-			kyma.Status.Conditions = append(kyma.Status.Conditions, newCondition)
+func (kyma *Kyma) InitModuleConditions() {
```

Comment:
After a second thought, I think I'll delete it, it designs to initialize all conditions, to have placeholder, but at the moment, we only have one condition, and it will be anyone create/update in `SyncConditionsWithModuleStates`, and I don't think later we will add more condition into it tbh.